pkg/macvlan/macvlan.go

```diff
@@ -314,6 +315,34 @@ func macvlanCmdAdd(args *skel.CmdArgs) error {
 	}
 
 	err = netns.Do(func(_ ns.NetNS) error {
+		// Get default interface
+		existingLink, err := netlink.LinkByName("eth0")
+		if err != nil {
+			return fmt.Errorf("couldn't get interface eth0: %v", err)
+		}
+
+		// Delete default route
+		existingDefaultRoute := netlink.Route{
```

Comment:
we are always sure that this `existingDefaultRoute` exists under all circumstances since we are pulling the link index from `existingLink` and you are handling the non-existing condition above. cool.